pylon/src/nginx.conf

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
```

Comment:
Test1

Reply:
Test 2

Reply:
Test 3

---
In reply to: [159610875](https://github.com/Microsoft/pai/pull/168#discussion_r159610875) [](ancestors = 159610875)

Reply:
test 4